events/events.go

```diff
@@ -0,0 +1,308 @@
+package events
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/service-fabrik-cli-plugin/errors"
+	"github.com/SAP/service-fabrik-cli-plugin/helper"
+	"github.com/cloudfoundry/cli/plugin"
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	httpClientDisabledSecurityCheck *http.Client
+	httpClientEnabledSecurityCheck  *http.Client
+)
```

Comment:
Why do we need 2 httpclients here?

Reply:
One http client is for connection with SSL and other is without. However, as of now, the client with SSL is not used, but if in future the requirement for same comes, then this code can easily take it up.